base/system/winlogon/lang/ro-RO.rc

```diff
@@ -36,7 +36,7 @@ CAPTION "Sistem de oprire"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_WARNING, IDC_SYSSHUTDOWNICON, 6, 7, 18, 20
-    LTEXT "Sistemul de oprire a fost inițializat. Salvați toate lucrările și închideți sesiunea. Toate lucrările nesalvate vor fi pierdute atunci când sistemul se va opri.", -1, 38, 7, 135, 40
+    LTEXT "Sistemul de oprire a fost inițializat. Salvați toate lucrările și închideți sesiunea. Toată munca nesalvată va fi pierdută atunci când sistemul se va opri.", -1, 38, 7, 135, 40
```

Comment:
I'm not keen on using "munca" here as it is usually used in the context of someone working a full-time job with a computer when that is not always the case. "Lucrările" is more of a generic term since you describe a wide range of activities, whatever you do on a computer. This could be an unsaved gaming session for example.

Reply:
What about now? I used the word "treaba".

Reply:
It would be good if you just kept `lucrările` as is, for the sake of consistency.

Reply:
I understand. I reverted the change.